Modules/arraymodule.c

```diff
@@ -868,6 +868,20 @@ array_slice(arrayobject *a, Py_ssize_t ilow, Py_ssize_t ihigh)
     return (PyObject *)np;
 }
 
+/*[clinic input]
+array.array.clear
+
+Remove all items from the array.
+[clinic start generated code]*/
+
+static PyObject *
+array_array_clear_impl(arrayobject *self)
+/*[clinic end generated code: output=5efe0417062210a9 input=5dffa30e94e717a4]*/
+{
+    if (array_resize(self, 0) == -1)
+        return NULL;
```

Comment:
Nit: according to the [PEP7](https://peps.python.org/pep-0007/#code-lay-out), braces are always required.
```suggestion
    if (array_resize(self, 0) == -1) {
        return NULL;
    }
```